scopes/design/ui/alert-card/alert-card.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { InfoAlertCard, WarningAlertCard, ErrorAlertCard, SuccessAlertCard } from './alert-card.composition';
```

Comment:
nice. please use:
```
expect(rendered).to.exist;
```

and add space between tests

Reply:
We are using React Testing Library not chai so to.exist isn't available. toBeTruthy() is available to us. Actually this test could be improved but we don't have all the types available yet. we are waiting for @GiladShoham to add all the types for testing library so we can use it fully. 
See docs for more info on testing library: https://testing-library.com/docs/